src/ms-store-badge.ts

```diff
@@ -50,7 +50,7 @@
   #imagesLocation = this.#env === "dev" ? "/images" : "https://getbadgecdn.azureedge.net/images";
   #platformDetails: PlatformDetails = { isWindows: false, windowsVersion: null, isEdgeBrowser: false };
 
-  static englishLanguage: SupportedLanguage = { name: "English", code: "en", imageSmall: { fileName: "English_S.png", }, imageLarge: { fileName: "English_L.png" }, imageLargeLight: {fileName: "English_LL.svg"} };
+  static englishLanguage: SupportedLanguage = { name: "English", code: "en-US", imageSmall: { fileName: "English_S.png", }, imageLarge: { fileName: "en-US dark.svg" }, imageLargeLight: {fileName: "en-US light.svg"} };
```

Comment:
We still have `imageSmall` here for backward compatibility with existing badges. Good.

But this PR deletes the small images, no?